templates/docs/patterns/container.md

```diff
@@ -0,0 +1,50 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Containers | Components
```

Comment:
I think it would be clearer if we call it "Section and block", this way the title will align with component names making it easier to understand and search for.

This would affect the heading on this page, file name (and therefore URL) as well.

And possibly in the code, the mixin instead of `vf-p-container` should be called `vf-p-section` (because there is no `.p-container`). I think it's fine for `vf-p-section` to include both `.p-section` and `.p-block` as they are closely related.